google/cloud/bigtable/CMakeLists.txt

```diff
@@ -50,6 +43,45 @@ include(CreateBazelConfig)
 # the client library
 add_library(
     google_cloud_cpp_bigtable # cmake-format: sort
+    admin/bigtable_instance_admin_client.cc
```

Comment:
Can you add this new directory (or at least the generated files) to the list of ignored files for code coverage purposes:

https://github.com/googleapis/google-cloud-cpp/blob/ffbe748b73a016e864f9a2b7bff371759adf70c5/.codecov.yml#L1

https://docs.codecov.com/docs/ignoring-paths

Reply:
The plan is to add integration tests for the admin APIs. I don't think that will get done this week due to the summit, so I will have ignore the files from coverage for now.